content/en/docs/collector/configuration.md

```diff
@@ -843,3 +871,23 @@ This creates two certificates:
   with the associated key in `cert-key.pem`.
 
 [dcc]: /docs/concepts/components/#collector
+
+## Override settings
+
+You can override Collector settings using the `--set` option. The settings you
+define with this method are merged into the final configuration after all
+`--config` sources are resolved and merged.
+
+The following examples show how to override settings inside nested sections:
+
+```shell
+# The following example sets the verbosity
+# level of the debug exporter to 'detailed'
+otelcol --set "exporters::debug::verbosity=detailed"
+# The following example overrides gRPC
+# settings for the OTLP receiver
+otelcol --set "receivers::otlp::protocols::grpc={endpoint:localhost:4317, compression: gzip}"
+```
```

Comment:
@bogdandrutu do you think having these examples make sense?

Reply:
If they're syntactically correct, could we merge and review later?

Reply:
:+1: let's do this

Reply:
Sweet! Could you approve? Thanks!